internal_transfers/actions/src/accounting.ts

```diff
@@ -62,24 +62,46 @@ export async function simulateSolverSolution(
     console.log(`batch ${transaction.hash} was not internalized.`);
     return null;
   }
-
-  const { full, reduced } = await simulateBoth(simulator, {
-    full: competition.fullCallData,
-    reduced: competition.reducedCallData,
-    common: {
-      contractAddress: SETTLEMENT_CONTRACT_ADDRESS,
-      sender: solverAddress,
-      value: "0",
-    },
-    startBlock: competition.simulationBlock,
-  });
-
-  return {
-    txHash: transaction.hash,
-    winningSettlement: competition,
-    full,
-    reduced,
-  };
+  // Try all blocks between claimed simulation block and mined block.
+  const numAttempts = transaction.blockNumber - competition.simulationBlock + 1;
```

Comment:
@olgafetisova - you were asking about this:

Did you see the comment above we attempt to simulate the transaction on every block from the claimed simulation block to actual mined transaction block.